pep/app/pods/application/route.ts

```diff
@@ -132,6 +136,119 @@ export default class Application extends PageLayout(Route.extend(ApplicationRout
+            }
+        });
+        this.tour.set('disableScroll', true);
+        this.tour.set('modal', true);
+        const steps: Step[] = [];
+        if (this.media.isMobile || this.media.isTablet) {
+            steps.push({
+                id: 'home',
+                attachTo: {
+                    element: '.navbar .navbar-toggler',
+                    on: 'bottom'
+                },
+
+                text: 'This button takes you home',
+                title: 'Home',
+                buttons: [
+                    {
+                        text: 'Next',
+                        type: 'next'
+                    }
+                ]
+            });
+        } else {
+            steps.push({
+                id: 'home',
+                attachTo: {
+                    element: '.nav-item.home',
+                    on: 'auto'
+                },
+                text: 'This button takes you home',
+                title: 'Home',
+                buttons: [
+                    {
+                        text: 'Next',
+                        type: 'next'
+                    }
+                ]
+            });
+        }
+        steps.push(
+            {
+                id: 'toggle-left',
+                attachTo: {
+                    element: '.sidebar-left .sidebar-toggle-handle',
+                    on: 'auto'
+                },
+                classes: 'tour-left-sidebar-spacing',
+                text: 'This button shows/hides the search',
+                title: 'Search',
+                buttons: [
+                    {
+                        text: 'Next',
+                        type: 'next'
+                    }
+                ],
+                scrollTo: true
+            },
+            {
+                id: 'toggle-right',
+                attachTo: {
+                    element: '.sidebar-right .sidebar-toggle-handle',
+                    on: 'left'
+                },
+                classes: 'tour-right-sidebar-spacing',
+                text: 'This button shows/hides the widgets',
+                title: 'Widgets',
+                buttons: [
+                    {
+                        text: 'Close',
+                        type: 'cancel'
+                    }
+                ],
+                when: {
+                    show: () => {
+                        this.currentUser.updatePrefs({ [PreferenceKey.TOUR_ENABLED]: false });
+                    }
+                }
+            }
+        );
+
+        await this.tour.addSteps(steps);
+        this.tour.start();
+    }
+
+    /**
+     * If the tour is enabled, show it
+     *
+     * @param {ApplicationController} controller
+     * @param {*} model
+     * @memberof Application
+     */
+    async setupController(controller: ApplicationController, model: any) {
+        super.setupController(controller, model);
+        if (this.currentUser.preferences?.tourEnabled) {
```

Comment:
should we skip executing this in fastboot? (even if the tour itself will stop itself from executing in fastboot, might be good to just short-circuit this here)